src/main/java/org/terasology/ligthandshadow/componentsystem/controllers/TeleporterSystem.java

```diff
@@ -103,6 +104,7 @@ private boolean isProperTeamSize(EntityRef teleporter, EntityRef player) {
         if (teleporterTeamCount - oppositeTeamCount < maxTeamSizeDifference) {
             if (teleporterTeamCount >= 0 && oppositeTeamCount >= 1 && !gameStart) {
                 sendEventToClients(TimerEvent::new);
+                player.send(new DelayedDeactivateBarrierEvent());
```

Comment:
If we call this a "delayed" event, we should encode the delay in this event, so that we only have a single source of truth how long the countdown is.